charts/federation-v2/charts/controllermanager/templates/role.yaml

```diff
@@ -12,31 +12,69 @@ rules:
 - apiGroups:
   - scheduling.federation.k8s.io
   resources:
-  - '*'
+  - replicaschedulingpreferences
   verbs:
-  - '*'
```

Comment:
Is `*` not equivalent to `get/watch/list/update`?

Reply:
No. It is not equivalent, e.g. it has no `create`. Because this is created by client but not controller.